.github/workflows/zap-scan.yml

```diff
@@ -0,0 +1,18 @@
+on:
+  schedule:
+    - cron:  '*/10 * * * *'
+
+jobs:
+  zap_scan:
+    runs-on: ubuntu-latest
+    name: Scan the webapplication
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: master
+      - name: ZAP Scan
+        uses: zaproxy/action-baseline@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          target: 'https://www.example.com/'
```

Comment:
shouldnt the target be https://www.zaproxy.org/ ??

Reply:
aw~ wrong branch, anyway updated this branch as well.